### ISSUE TITLE
Check if already-forked feedstocks have open pulls

### ISSUE THREAD

**Issue by pmlandwehr** [pmlandwehr/tick-feedstocks#10]
If a feedstock already has a fork and an open pull is in progress, we can back off with versioning work. (Conceivably, it might be faster to cull packages from the upgrade process by checking for these conflicts earlier.)

**Comment by jakirkham** [comment]
Would be worthwhile to take a look at the other `conda-execute` scripts at `conda-forge`. They have a way to solve this. Namely `pin_the_slow_way.py` and `regenerate_feedstock.py`.

**Comment by pmlandwehr** [comment]
Yeah - I've shamelessly ripped of `regenerate_feedstock.py` for some of this already, lots of good code to use there.
